.ado/azure-pipelines.yml

```diff
@@ -146,3 +146,43 @@ jobs:
           publishJUnitResults: false
           javaHomeOption: 'JDKVersion'
           sonarQubeRunAnalysis: false
+
+  - job: NuGetPublishDryRun
```

Comment:
How often is the publish pipeline breaking due to someone's changes? I'm wondering if a failed publish merits blocking checkins if the publishes failing are commonly due to external factors

Reply:
Usually one of us forgets to update the .nuspec when we make a new native module, or we might put the paths in the nuspec wrong. At least one time the token to publish expired, which you know of because you helped fix it.
Once it was because we tried to integrate rex-kit and changed the bundles from .jsbundle to .bundle.
Most of the time it's Apple Engineers changing something on the apple side, not external.


Also, I would be happy to have this just *publish* the NuGet too because having a NuGet from a WIP PR is helping for testing downstream. I think that would block others working on JS only stuff if the NuGet pipeline broke again because of expired certs or the like though.